packages/x-components/src/x-modules/facets-next/components/lists/filters-injection.mixin.ts

```diff
@@ -0,0 +1,72 @@
+import { Filter } from '@empathyco/x-types-next';
```

Comment:
'@empathyco/x-types-next' import is duplicated.

```suggestion
import { Filter, HierarchicalFilter, isHierarchicalFilter } from '@empathyco/x-types-next';
```

- [ ] Update `@empathyco/x-types-next` import